src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -25,6 +25,7 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class ScriptJwtBearerExtensions
     {
+        private const string SlotSeparator = "__";
```

Comment:
nit: a little extra information on what "slot" is

```suggestion
        private const string DeploymentSlotSeparator = "__";
```